### PR TITLE
Mount 4.4 jobs into all components

### PR DESCRIPTION
**Pull request by stevekuznetsov**
Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @openshift/openshift-team-developer-productivity-test-platform

### CODE REVIEW

File: core-services/prow/03_deployment/tot.yaml [hunk 1]
```diff
@@ -86,11 +86,14 @@ items:
             mountPath: /etc/job-config/4.1
             readOnly: true
           - name: job-config-42
-            mountPath: /etc/job-config/4.2
+            mountPath: /etc/job-config/4.2w
```
Comment (hongkailiu) [inline core-services/prow/03_deployment/tot.yaml]: /hold
typo?

Comment (stevekuznetsov) [reply core-services/prow/03_deployment/tot.yaml]: yikes

Comment (stevekuznetsov) [reply core-services/prow/03_deployment/tot.yaml]: fixed

Comment (hongkailiu) [reply core-services/prow/03_deployment/tot.yaml]: /hold cancel

**Pull request merged**